docs/examples.rst

```diff
@@ -0,0 +1,285 @@
+.. _examples:
+
+Examples
+=========
+
+.. contents::
+   :local:
+
+.. _auth-examples:
+
+Authentication: ``dds auth``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. _login-example:
+
+Start authenticated session ("Log in")
+---------------------------------------
+
+After running the command ``dds auth login``, you will be prompted to fill in information in the following order:
+
+1. Your DDS *username*
+2. Your DDS *password*
+   
+   .. admonition:: The password is hidden
+    
+        Note that the password *will not be printed out* when you type it; The password is hidden for security purposes.
+
+3. A one time code.
+   
+   .. admonition:: Email is default
+
+        If you have not configure the 2FA method (see section :ref:`below<2fa-config-example>`), a one time code is sent to your email. If you have set the 2FA method to *Authenticator App*, the one-time code will be shown in that app.
+
+.. image:: ../img/dds-auth-login.svg
+
+
+.. admonition:: Forgotten your...
+
+    * **Username?** Contact support. Changing username or authenticating with email is currently not possible.
+    * **Password?** You can reset your password `here <https://delivery.scilifelab.se/reset_password>`_.
+
+.. danger:: 
+
+    After completing authentication, ``dds-cli`` will automatically save an authentication in the directory where you run the command, unless otherwise specified (see the command documentation). ``dds-cli`` will use this token as a session when you run future commands. 
```

Comment:
`... will automatically save an authentication token in the directory where you run the command ...`

and isn't the token created in the home directory?

Reply:
> don't you think it is better to add `token file` here?

Was this message actually suppose to be sent yesterday or was it a case of browser cache again? 😅

Reply:
it was a leftover from before you added the token file text :)